projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/Utils.kt

```diff
@@ -161,17 +166,27 @@ private const val SUBSYSTEM = "PROJECTOR_SERVICE_CONFIG"
 const val PROJECTOR_RW_TOKEN_KEY = "PROJECTOR_RW_TOKEN"
 const val PROJECTOR_RO_TOKEN_KEY = "PROJECTOR_RO_TOKEN"
 
-fun loadToken(key: String): String? {
+fun safeLoad(key: String): String? {
   val credentialAttributes = createCredentialAttributes(key)
   return PasswordSafe.instance.getPassword(credentialAttributes)
 }
 
-fun storeToken(key: String, value: String?) {
+fun safeStore(key: String, value: String?) {
   val attributes = createCredentialAttributes(key)
   val credentials = Credentials(user = key, password = value)
   PasswordSafe.instance.set(attributes, credentials)
 }
 
+const val RANDOM_PASSWORD_LEN = 11
+
+fun generatePassword(): String {
+  val charPool: List<Char> = ('a'..'z') + ('A'..'Z') + ('0'..'9')
+  return (1..RANDOM_PASSWORD_LEN)
+    .map { Random.nextInt(0, charPool.size) }
+    .map(charPool::get)
+    .joinToString("")
+}
```

Comment:
Was before this PR, but can be improved now:

```suggestion
fun generatePassword(): String {
  val charPool = ('a'..'z') + ('A'..'Z') + ('0'..'9')
  return generateSequence { charPool.random() }
    .take(RANDOM_PASSWORD_LEN)
    .joinToString("")
}
```